### PR TITLE
feat: disable activity workers

### PR DESCRIPTION
**Pull request by cv65kr**
# Reason for This PR

https://github.com/roadrunner-server/roadrunner/issues/2089

## Description of Changes

Disable activity workers

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the MIT license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [ ] All commits in this PR are signed (`git commit -s`).
- [ ] The reason for this PR is clearly provided (issue no. or explanation).
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.

### CODE REVIEW

**Comment by cv65kr** [comment]
Changes need to be provided to https://github.com/roadrunner-server/pool/blob/master/pool/config.go#L42

**Comment by rustatian** [comment]
Hey @cv65kr 👋🏻 Thank you for the PR 👍🏻 
> Changes need to be provided to

To skip workers allocation, after the initializing the pool in the config.go, check if that option to disable activity workers is active (set to true) and set `NumWorkers` to 0: [link](https://github.com/temporalio/roadrunner-temporal/blob/master/config.go#L100).

File: aggregatedpool/workers.go [hunk 1]
```diff
@@ -86,6 +86,10 @@ func TemporalWorkers(wDef *Workflow, actDef *Activity, wi []*internal.WorkerInfo
 		}
 
 		for j := 0; j < len(wi[i].Activities); j++ {
```
Comment (rustatian) [inline aggregatedpool/workers.go:88]: You may check for the `disableActivityWorkers` before the loop and use goto <some-var> to jump directly to the `workers = append(workers, wrk)` to skip useless iteration.

File: internal.go [hunk 2]
```diff
@@ -24,6 +24,9 @@ const (
 
 func (p *Plugin) initPool() error {
 	var err error
+	if p.config.DisableActivityWorkers {
```
Comment (rustatian) [inline internal.go:27]: No need to do that here, see comment about config.go.

**Comment by rustatian** [comment]
We can't change the default behavior for the pool.

In the pool (`NewPool`), last arg are options. You may introduce a new option, like, `WithOverrideWorkersNum` (something like that) and pass this option in the pool in the rr-temporal plugin. Options are applied after `InitDefaults` and before actual workers allocation.

**Review by rustatian — APPROVED** [review]

**Review by rustatian — APPROVED** [review]
LGTM, thanks @cv65kr 👍🏻

**Pull request merged**